### ISSUE TITLE
Date missing in order printable invoice

### ISSUE THREAD

**Issue by mikemurray** [reactioncommerce/reaction#3815]
### Expected behavior

Should see a date in printable invoice view

### Actual behavior

Error in browser console and no date in printable invoice view

![localhost_4000_dashboard_pdf_orders_sgfpvapppeeeb9dy9](https://user-images.githubusercontent.com/42217/36509425-926d9eec-1714-11e8-8496-bff4070f310b.png)

```
Exception in template helper: TypeError: Cannot read property 'locale' of undefined
    at Object.dateFormat (http://localhost:4000/app/app.js?hash=cefd505fb113f9b6646f231979b33b6bcdd4462d:46308:12)
    at http://localhost:4000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3051:16
    at http://localhost:4000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:1715:16
    at http://localhost:4000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3103:66
    at Function.Template._withTemplateInstanceFunc (http://localhost:4000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3744:12)
    at http://localhost:4000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3102:27
    at Spacebars.call (http://localhost:4000/packages/spacebars.js?hash=547cf8e466d1d52603d19bd5f48fb5df184fd237:172:18)
    at Spacebars.mustacheImpl (http://localhost:4000/packages/spacebars.js?hash=547cf8e466d1d52603d19bd5f48fb5df184fd237:106:25)
    at Object.Spacebars.mustache (http://localhost:4000/packages/spacebars.js?hash=547cf8e466d1d52603d19bd5f48fb5df184fd237:110:39)
    at Blaze.View._render (http://localhost:4000/app/app.js?hash=cefd505fb113f9b6646f231979b33b6bcdd4462d:45803:24)
```

### Steps to reproduce the behavior

1. Add products to cart and checkout
2. As an admin, approve and capture an order
3. Click the "Print Invoice"  button

![basic_reaction_product](https://user-images.githubusercontent.com/42217/36510456-e8d211ca-1717-11e8-8aae-4359de959ff5.png)

### Versions

Node: 7.2.0
NPM: 5.6.0
Meteor Node: 8.9.4
Meteor NPM: 5.6.0
Reaction CLI: 0.28.0
Reaction: 1.8.0
Reaction branch: release-1.8.0
Docker: 17.12.0-ce

**Comment by zenweasel** [comment]
This is a very recent regression as I remember specifically testing this for the moment PR

**Comment by zenweasel** [comment]
Closed via #3816
